database/seeders/DeliveryStatusSeeder.php

```diff
@@ -12,6 +14,14 @@ class DeliveryStatusSeeder extends Seeder
      */
     public function run(): void
     {
-        //
+        DeliveryStatus::create([
+            "status"=> "Pending"
+        ]);
+        DeliveryStatus::create([
+            "status"=> "Completed"
+        ]);
+        DeliveryStatus::create([
+            "status"=> "Canceled"
+        ]);
```

Comment:
Als je een enum gebruikt moet je hier de **niet** gelocaliseerde waardes toevoegen aan de database en uiteindelijk in je views de gelocaliseerde.